### ISSUE TITLE
increase buffer size for syslog-ng

### ISSUE THREAD

**Issue by soyacz** [scylladb/scylla-cluster-tests#6689]
During network issues or high traffic syslog-ng buffers fill up and drop messages.

Let's try increasing syslogng buffers to tackle this issue.

**Comment by fruch** [comment]
We try to guesstimate how much logs scylla can generate within 30sec, and based on that set the size

question is how we should do, taking one 12h job, and analyze the logs timestamps ? to find the size ?

**Comment by roydahan** [comment]
First, I would check with @kbr-scylla what is the scenario that reproduces for him consistently the drop of syslog-ng.
Then, with that I would start by doubling the buffer and see if it solve the issue.
Then, I would test it on one of the MV or LWT longevities that we have and spam the logs and see if it causes any issue.

**Comment by kbr-scylla** [comment]
> First, I would check with @kbr-scylla what is the scenario that reproduces for him consistently the drop of syslog-ng.

https://jenkins.scylladb.com/job/scylla-staging/job/KamilBraun/job/longevity-schema-changes-3h-test-artsiom_mishuta-clone/

I checked some of the recent runs, and they had the problem.
(I didn't check early runs, but I would guess they also had this problem)


> We try to guesstimate how much logs scylla can generate within 30sec, and based on that set the size

30s is pretty conservative. Why so low? We're already keeping the entire Scylla log in system.log. Having a separate buffer that is just a fraction of that should not be a problem. Even if we picked e.g. 10 minutes it would still be a small fraction of system.log during longevity test.

**Comment by kbr-scylla** [comment]
https://jenkins.scylladb.com/job/scylla-staging/job/KamilBraun/job/longevity-schema-changes-3h-test-artsiom_mishuta-clone/16/
https://cloudius-jenkins-test.s3.amazonaws.com/9f0e827b-8a9a-4767-b4b4-0582d3f9415f/20231007_113814/db-cluster-9f0e827b.tar.gz   

Example of event from longevity-ParallelAddDropColumnMonk-db-node-9f0e827b-1/messages.log:
```
2023-10-07T10:46:10.432+00:00 longevity-ParallelAddDropColumnMonk-db-node-9f0e827b-1      !ERR | syslog-ng[4611]: EOF occurred while idle; fd='39'
2023-10-07T10:46:10.432+00:00 longevity-ParallelAddDropColumnMonk-db-node-9f0e827b-1   !NOTICE | syslog-ng[4611]: Syslog connection closed; fd='39', server='AF_INET(10.12.3.178:32768)', time_reopen='60'
```

Unfortunately the timestamps around it don't tell how long the event took. But we can cross-check with system.log which is supposed to use Scylla timestamps.

From messages.log, before the event:
```
2023-10-07T10:45:10.431+00:00 longevity-ParallelAddDropColumnMonk-db-node-9f0e827b-1    !DEBUG | scylla[5312]:  [shard 0:stre] raft - [43005297-053a-4cfa-9d73-bdb80f30fa84] io_fiber starting iteration 5529
```
after the event:
```
2023-10-07T10:46:10.432+00:00 longevity-ParallelAddDropColumnMonk-db-node-9f0e827b-1    !DEBUG | scylla[5312]:  [shard 0:stre] raft - [43005297-053a-4cfa-9d73-bdb80f30fa84] io_fiber starting iteration 6842
```

and the corresponding logs in system.log:
```
Oct 07 10:44:13 longevity-ParallelAddDropColumnMonk-db-node-9f0e827b-1 scylla[5312]:  [shard 0:stre] raft - [43005297-053a-4cfa-9d73-bdb80f30fa84] io_fiber starting iteration 5529
```
and
```
Oct 07 10:45:10 longevity-ParallelAddDropColumnMonk-db-node-9f0e827b-1 scylla[5312]:  [shard 0:stre] raft - [43005297-053a-4cfa-9d73-bdb80f30fa84] io_fiber starting iteration 6842
```

so it was around a minute.

I checked the size of the log file when I cut everything before `iteration 5529` and after `iteration 6842`. The result was
```
$ du -sh tmp.log 
6.8M	tmp.log
```

**Comment by kbr-scylla** [comment]
Even if we used a buffer size of, let's say, 100MB, it would be nothing compared to the available disk sizes, or the amount of data that Scylla stores during longevity tests.
